app/models/campaign.rb

```diff
@@ -46,7 +46,8 @@ class Campaign < ActiveRecord::Base
 
   # If there are overlapping campaigns, returns the one furthest out.  
   def self.current_campaign_id(org_id)
-    joined_by(org_id).order(donation_deadline: "DESC").first.id
+    last_campaign = joined_by(org_id).order(donation_deadline: "DESC").first
+    last_campaign ? last_campaign.id : -1
```

Comment:
Related to comment in the `organizations_controller.rb` re: returning `nil` vs `-1`

I think it would be more rails-like to a) return nil instead of -1; b) return the campaign, rather than just the ID (when it's not nil). Also, I made a comment earlier about moving this into the `Organization` model, what's the difference between these two blocks of code?

```
o = Organization.find(1)
o.campaigns
```

vs

```
Campaign.joined_by(1)
```